### DISCUSSION TITLE
[Office Hours] Syncing VSCode with XCode for Preview and adding video play

### DISCUSSION THREAD

**Discussion by christinaba24** [CS342/.github#60] [Office Hours]
### Related Discussion

N/A

### Reproduction

N/A

### Additional context

Working on midterm presentation, possible to meet on Zoom?

### Code of Conduct

- [X] I agree to follow this course's Code of Conduct and Contributing Guidelines

**Comment by PSchmiedmayer** [comment]
@christinaba24 Has this issue been resolved and if so, can you share some more context in the initial discussion post about the issue and how you resolved it in a short follow up post here?
